grails-app/controllers/org/pih/warehouse/product/ProductController.groovy

```diff
@@ -938,7 +938,7 @@ class ProductController {
                 flash.message = "All ${command?.products?.size()} product(s) were imported successfully."
                 redirect(controller: "product", action: "importAsCsv", params: [tag: tags[0]])
             } catch (ValidationException e) {
-                command.errors = e.errors
+                command.errors.reject(e.message)
```

Comment:
I don't know if this is what we want to do. Can you provide justification for this? The validation exception has an Errors object that includes an detailed error message for each invalid data element. Displaying just the message associated with the validation exception give us a generic error message without those details.

Reply:
for some reason, that I wasn't able to figure out, assigning `e.errors` to `command.errors` didn't work.
It didn't display any error messages on `.gsp` page even thought error was was thrown.

I will try to do `product.errors.reject("...")` inseatd of throwing an exception, I was just following an example where few line below we have
```
 if (!product.save(flush: true)) {
    hrow new ValidationException("Could not save product '" + product.name + "'", product.errors)
}
```

Reply:
it was `importAsCsv.gsp` 
https://github.com/openboxes/openboxes/blob/develop/grails-app/views/product/importAsCsv.gsp#L18
even thought it has appropriate logic for checking if `command` has errors and siplay those errors but for some reason it didn't as if it lost those errors somewhere along the way

Reply:
Ok, well make the suggested change(s) and if you still can't get it to work I can take a look.